src/build/__main__.py

```diff
@@ -117,7 +114,12 @@ def main_parser():  # type: () -> argparse.ArgumentParser
         action='store_true',
         help='build a wheel',
     )
-    parser.add_argument('--outdir', '-o', metavar='dist', type=str, default=out, help='output directory')
+    parser.add_argument(
+        '--outdir',
+        '-o',
+        type=str,
```

Comment:
Please don't remove the metavar.

Reply:
I removed it because `--help` would print `--outdir dist, -o dist` which I thought was confusing / better server by the help text.  Should I add it back as 'dist'?

Reply:
maybe then make metavar just ``dir``, showing needs to be a directory?

Reply:
It prints out 'OUTDIR' by default though I suppose that might be a little repetitive?

Reply:
I'll take `  --outdir dir, -o dir` over `  --outdir OUTDIR, -o OUTDIR`, though maybe `  --outdir dest, -o dest` or `--outdir to, -o to` is better.

Reply:
Topics are usually marked resolved by the change requester. It's their way of saying they accept the change.

Reply:
Apologies.